src/main/java/org/dinosaur/foodbowl/domain/store/domain/vo/SchoolName.java

```diff
@@ -0,0 +1,36 @@
+package org.dinosaur.foodbowl.domain.store.domain.vo;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import jakarta.validation.constraints.NotNull;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.dinosaur.foodbowl.domain.store.exception.SchoolExceptionType;
+import org.dinosaur.foodbowl.global.exception.BadRequestException;
+
+@Getter
+@Embeddable
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@Embeddable
```
조금 더 중요한 `@Embeddable`을 밑으로 내리는건 어떨까요?!